synapse/config/repository.py

```diff
@@ -189,6 +173,8 @@ def read_config(self, config, **kwargs):
                     "to work"
                 )
 
+            from netaddr import IPSet
```

Comment:
why is this no longer protected with a `try/except` ?

Reply:
The previous `try/except` only defended against import errors but netaddr is a dependency included in the "url_preview" feature. So we can be guaranteed that it's available for import. I can see that's not clear though so I'll add a comment.

Reply:
ahh sorry I failed to read what was happening here properly.